app/models/category.rb

```diff
@@ -8,4 +8,17 @@ class Category < ApplicationRecord
   def self.all_for_nav
     @all_for_nav ||= Category.all
   end
+
+  def self.items_in_category
+    categories = Category.select('categories.id, count(items.id) as quantity').
+                  left_outer_joins(:items).
+                  group(:id)
```

Comment:
This is a pretty cool sql query. I think we can look at method refactoring down the line.